recipe/run_test.sh

```diff
@@ -8,6 +8,7 @@ set -x
 # Testing QGIS command currently is found as lint in meta.yaml
 # QGIS has no --version, and --help exists 2
 qgis --help || [[ "$?" == "2" ]]
+qgis_process --version
```

Comment:
try setting `QT_QPA_PLATFORM=offscreen` first to see if this fixes Linux...

Reply:
That did the trick!